src/udev/udevadm-info.c

```diff
@@ -40,6 +41,8 @@ typedef enum QueryType {
 static bool arg_root = false;
 static bool arg_export = false;
 static const char *arg_export_prefix = NULL;
+static bool arg_wait_for_initialization = false;
```

Comment:
I don't think a separate `bool` is needed. Just initialize `arg_wait_for_initialization` to 0.